updating styles (#119)
infinum/eightshift-docs: website/docs/basics/tips-tricks.md

---
id: tips-tricks
title: Tips & Tricks
---

> As we are making our projects using this setup we are going to share our tips and tricks for faster development, easier debugging and some cool ideas we had along the way.

Reviewer: ```suggestion
Since we are creating our projects using this setup, we are going to share our tips and tricks for faster development, easier debugging and some cool ideas we had along the way.
```

### Always disable cache when you are writing block editor components

> When you are writing an JavaScript part of the block you should always have your browser inspector open and checkbox checked for disabling browser cache in the network tab. WordPress core is caching JS files and you may not always get the latest changes fo you code in the editor. This way you can be sure that everything is up to date.

Reviewer: ```suggestion
When you are writing a JavaScript part of the block you should always have your browser inspector open and checkbox checked for disabling browser cache in the network tab. WordPress core is caching JS files and you may not always get the latest changes of your code in the editor. This way you can be sure that everything is up to date.
```

### When to restart your Webpack watch?

> As we described in the previous chapters Webpack watch is used to monitor all your code changes and update the compiled files with your latest changes. We have set our folder structure in a way so we can have files automatically included in the build. Here are some of the times you must restart you Webpack watch:

Reviewer: ```suggestion
As we described in the previous chapters Webpack watch is used to monitor all your code changes and update the compiled files with your latest changes. We have set our folder structure in a way so we can have files automatically included in the build. However, there are some times you must restart your Webpack watch:
```

* Adding a new block/component.
* Changing the block/component folder name.
* Changing any of the file names in the block/component folder.
* Adding/removing a JS package to the project.
* Changing the webpack, babel, eslint, stylelint configuration.
* Changing the project domain name for Browser sync.
* Adding/removing a new JS or SCSS file in the global assets folder.

### Using boilerplate and boilerplate-plugin in the same project

When you are using one boilerplate for a theme and another for a plugin keep in mind that you will have to do some changes. We would recommend leaving the theme as is and making changes to the plugin. Here are some of the changes you should make:

* If using manifest, you should update the `MANIFEST_ITEM` filter name to something else.

> * If using blocks, change the default category all you plugin blocks are stored in.

Reviewer: ```suggestion
* If using blocks, change the default category of all your plugin blocks.
```

You meant this?

> * If using blocks, update all blocks to a new category in the each block manifest.json file.

Reviewer: ```suggestion
* If using blocks, update all blocks to a new category in each block `manifest.json` file.
```

> * If using blocks, update the block namespace in the global block manifest.json file.

Reviewer: ```suggestion
* If using blocks, update the block namespace in the global block `manifest.json` file.
```

> * If using blocks, change the blocks background and foreground color (Not necessary but it would be a good UX).

Reviewer: ```suggestion
* If using blocks, change the blocks' background and foreground color (Not necessary but it would be a good UX).
```
